src/settings/lib/color-palette.scss

```diff
@@ -1,3 +1,44 @@
-$tbds-brand-red: #e03131 !default;
-$tbds-brand-gray: #29292c !default;
-$tbds-brand-blue: #0b758c !default;
+$tbds-color-ralph-00: #fff9f9 !default;
+$tbds-color-ralph-10: #f5b1b1 !default;
+$tbds-color-ralph-20: #ea6f6f !default;
+$tbds-color-ralph-30: #e03131 !default;
+$tbds-color-ralph-40: #a70707 !default;
+$tbds-color-ralph-50: #6e0000 !default;
+
+$tbds-color-rhonda-00: #f7fcfc !default;
+$tbds-color-rhonda-10: #92cbd7 !default;
+$tbds-color-rhonda-20: #439eb1 !default;
+$tbds-color-rhonda-30: #0b758c !default;
+$tbds-color-rhonda-40: #22545e !default;
+$tbds-color-rhonda-50: #202e30 !default;
```

Comment:
@tysongach I am tinkering with a 6 step concept here. I think this leaves us with just enough useful flexibility. Here are some screenshots to visualize this:

![Screen Shot 2019-10-04 at 4 38 59 PM](https://user-images.githubusercontent.com/570184/66238371-807afb80-e6c5-11e9-9e5a-81ff0854982e.png)
![Screen Shot 2019-10-04 at 4 38 46 PM](https://user-images.githubusercontent.com/570184/66238373-8375ec00-e6c5-11e9-968b-c70b05bf83e6.png)